webpack.common.js

```diff
@@ -44,6 +44,7 @@ const config = {
             "bourbon": "bourbon.scss",
             "plotly-basic": "plotly.js-basic-dist",
             "plotly-gl2d": "plotly.js-gl2d-dist",
+            "availableTags": path.join(__dirname, "example/tags/tags.json"),
```

Comment:
See previous comments. Let's make the example tags an example plugin that loads the JSON file and registers the tags with the API.